inference-engine/ie_bridges/python/src/openvino/offline_transformations/CMakeLists.txt

```diff
@@ -1,64 +0,0 @@
-# Copyright (C) 2018-2021 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-#
-
-set(TARGET_NAME "offline_transformations_api")
```

Comment:
'model_optimizer' target still depends on 'offline_transformations_api' and now always fails. Can you please fix it?